tests/tests_integration/test_api/test_simulators/seed/data.py

```diff
@@ -232,3 +235,88 @@
     "name": "Simulator Routine - Test",
     "description": "Simulator Routine - Description Test",
 }
+
+
+simulator_routine_revision = {
+    "externalId": None,
+    "routineExternalId": resource_names["simulator_routine_external_id"],
+    "configuration": {
+        "schedule": {"enabled": True, "cronExpression": "*/10 * * * *"},
```

Comment:
Again, the service should ensure some kind of jitter/random delay for these cron jobs